test/unit/gateways/merchant_e_solutions_test.rb

```diff
@@ -19,7 +19,10 @@ def setup
       recurring_pmt_num: 11,
       recurring_pmt_count: 10,
       billing_address: address,
-      description: 'Store Purchase'
+      description: 'Store Purchase',
+      card_on_file: 'Y',
+      cit_mit_indicator: 'C101',
+      account_data_source: 'Y'
```

Comment:
If nothing else, we should add in some tests to assert that these values are being added to the post body.

Reply:
Added @naashton